db/models/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var fs        = require('fs');
```

Comment:
How do you feel that in new (maybe next?) PR that we could use babel here so we can use ES2015+?